src/Integration/LocalServices/CredentialStore.cs

```diff
@@ -28,6 +29,7 @@ namespace SonarLint.VisualStudio.Integration
     /// <summary>
     /// Simple wrapper around SecretStore to allow it to be registered as a ILocalService
     /// </summary>
+    [Export(typeof(ICredentialStoreService))]
```

Comment:
Prep work to be able to MEF-import IConfigurationProvider, which depends on SolutionBindingDataReader -> SolutionBindingCredentialsLoader.
The rest of the classes could then be new-ed.